uuid/example_test.go

```diff
@@ -0,0 +1,55 @@
+// To enable support of UUID in msgpack with
+// [google/uuid](https://github.com/google/uuid), import tarantool/uuid
+// submodule.
+//
+// Run Tarantool instance before example execution:
+// $ cd uuid
+// $ TT_LISTEN=3013 TT_WORK_DIR=$(mktemp -d -t 'tarantool.XXX') tarantool config.lua
```

Comment:
Whole guide is consistent: we have `TEST_TNT_` vars to control work dir and listen config. Either we should get rid of `TEST_TNT_*` (and I don't think it would work nice since we don't set them all at first box.cfg to ensure every other thing is bootstrapped) or do not use `TT_` vars.

Reply:
Tarantool allows passing box.cfg options via `TT_<NAME>` env variables. Command-line provided in a comment works.

Reply:
I still do not see any reason to create inconsistency

Reply:
Replaced `TT_<NAME>` with `TEST_TNT_<NAME>`.

Reply:
(Everywhere)